pkg/auth/providers/activedirectory/activedirectory_client.go

```diff
@@ -411,9 +407,7 @@ func (p *adProvider) getPrincipal(distinguishedName string, scope string, config
 	return principal, nil
 }
 
-func (p *adProvider) searchPrincipals(name, principalType string, config *v32.ActiveDirectoryConfig, lConn *ldapv3.Conn) ([]v3.Principal, error) {
-	name = ldapv3.EscapeFilter(name)
```

Comment:
Note: this is now done now when building the query in `searchUser` and `searchGroup`.